mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Postgrex.Mixfile do
   def project do
     [app: :postgrex,
      version: "0.9.2-dev",
-     elixir: "~> 1.0",
+     elixir: "~> 1.1",
```

Comment:
Sounds like a good idea. Honestly, we can just call GenServer.whereis unless we have a process.